src/Icons/doc/index.rst

```diff
@@ -541,30 +541,30 @@ Full Configuration
 
     # config/packages/ux_icons.yaml
     ux_icons:
-        # The local directory where icons are stored.
+        # The local directory where icons are stored
         icon_dir: '%kernel.project_dir%/assets/icons'
 
         # Default attributes to add to all icons.
         default_icon_attributes:
-            # Default:
             fill: currentColor
+            'aria-hidden': true
```

Comment:
```suggestion
            # 'aria-hidden': true # it is always added automatically unless 'aria-label', 'aria-labelledby' or 'title' is provided
```

Reply:
You are right. Using `aria-hidden` can be confusing. I changed it for `'font-size': '1.25em'`